src/main/java/frc/robot/controllers/TalonSRXDrivetrain.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.controllers;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Encoder;
+
+import frc.robot.util.*;
+
+public class TalonSRXDrivetrain extends Drivetrain {
+    public TalonSRX leftMotor1, leftMotor2, rightMotor1, rightMotor2;
+    public Encoder leftEncoder, rightEncoder;
+    private static PID leftDrivePID = new PID(1.0, 0, 0);
+    private static PID rightDrivePID = new PID(1.0, 0, 0);
```

Comment:
The same question about static as above. 
Btw, it would be nice to read PID parameters from network tables to save time while tuning the robot. (1.0, 0, 0) could became just defaults, in case there are no corresponding values in the network tables.